### ISSUE TITLE
Crystal 0.11 support

### ISSUE THREAD

**Issue by sdogruyol** [waterlink/active_record.cr#35]
Hey @waterlink 

Seems like 0.11 doesn't support `abstract def` on metaclasses. Are you still supporting this library?

Here's the stacktrace

```
Error in ./src/adapter.cr:22: can't define abstract def on metaclass

    abstract def self.build(table_name, primary_field, fields, register = true)
```

**Comment by waterlink** [comment]
Hi Serdar, 

Thank you for finding this problem! 

Yes, of course, I still support it. I simply didn't use it for some time and, therefore, didn't notice this problem. 

I will take a look into your PR, and why it fails in travis in an hour or so.

Kind Regards, 
Oleksii

**Comment by waterlink** [comment]
I have released 0.1.3 with the fixes.

**Comment by sdogruyol** [comment]
Great job!
